src/azul/service/elasticsearch_service.py

```diff
@@ -93,6 +95,9 @@ def _translate_filters(self, filters: Filters, field_mapping: JSON) -> MutableFi
             assert isinstance(value, dict)
             assert isinstance(one(value.values()), list)
             field_type = self.field_type(tuple(key.split('.')))
+            if consolidate_facet:
```

Comment:
```suggestion
            if key == 'organismAge':
```
Is the `consolidate_facet` boolean variable needed?